test/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys, types
+sys.modules['pandas'] = types.ModuleType('pandas', 'Fake pandas module')
```

Comment:
All tests pass and everything looks good, but can you explain why you're mocking out the pandas module? Even if it isn't required right now, shouldn't we still test against pandas?